apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -615,9 +615,9 @@ export const notifications = {
         'Fyrirsögn í pósti til verjanda þegar hann er skráður á mál.',
     },
     body: {
-      id: 'judicial.system.backend:notifications.defender_assigned_email.body_v3',
+      id: 'judicial.system.backend:notifications.defender_assigned_email.body_v4',
       defaultMessage:
-        '{court} hefur skráð þig verjanda í máli {courtCaseNumber}.<br /><br />{defenderHasAccessToRVG, select, true {Gögn málsins eru aðgengileg á {linkStart}yfirlitssíðu málsins í Réttarvörslugátt{linkEnd}} other {Þú getur nálgast gögn málsins hjá {courtName} ef þau hafa ekki þegar verið afhent}}.',
+        '{court} hefur skráð þig {advocateType, select, LAWYER {lögmann einkaréttarkröfuhafa} LEGAL_RIGHTS_PROTECTOR {réttargæslumann einkaréttarkröfuhafa} other {verjanda}} í máli {courtCaseNumber}.<br /><br />{defenderHasAccessToRVG, select, true {Gögn málsins eru aðgengileg á {linkStart}yfirlitssíðu málsins í Réttarvörslugátt{linkEnd}} other {Þú getur nálgast gögn málsins hjá {courtName} ef þau hafa ekki þegar verið afhent}}.',
```

Comment:
Similar to the comment above, wouldn't the second text also be wrong for the civil claimant, or can he get the case files from the court if he wishes?